### ISSUE TITLE
As a Liaison I want a Bookings dashboard so I can manage approvals through the app.

### ISSUE THREAD

**Issue by nopivnick** [ITPNYU/booking-app#346]

**Comment by lucia-gomez** [comment]
Made a few assumptions here, @nopivnick could you check them?
- Liaisons should only see bookings with status `REQUESTED`
- Liaisons should only see bookings for their department, and one person could be liaison for multiple departments
- Liaisons don't want to filter by date, they should see all bookings currently pending their approval
- Admins can also see the Liaison dashboard

![Image](https://github.com/user-attachments/assets/49804386-348e-4258-9f04-51f7b5d7eda0)

**Comment by nopivnick** [comment]
@lucia-gomez those all sound like good assumptions to me.

**Comment by nopivnick** [comment]
@lucia-gomez I can see a request for a department other than ITP/IMA (for which I am the liaison). is that because I'm an Admin as well?

![Image](https://github.com/user-attachments/assets/4e6f4118-e030-478d-8b49-ec545bfcabbe)

**Comment by lucia-gomez** [comment]
yes that's because you're an admin. should that not be the case?

**Comment by nopivnick** [comment]
@lucia-gomez I think the Liaison context should only display requests for the Liaison's own department, regardless whether the Liaison also happens to be an Admin.

**Comment by lucia-gomez** [comment]
actually i see you were on development for that previous screenshot. You aren't listed as a liaison on dev. I am (as a test) and I only see bookings with status REQUESTED for the department of which i am a liaison
